### ISSUE TITLE
Inference on KITTI

### ISSUE THREAD

**Issue by jtang10** [amoudgl/pygoturn#27]
Hello there, I'm a student working on system level characterization of deep learning tasks and want to use GOTURN as an example for object tracking. I found your codebase is very concise and easy to understand. However, I'm not very familiar with object tracking task, so I want to know

1. Can your pretrained model directly on KITTI tracking data without fine-tuning? I don't really need the inference accuracy to be optimal as long as it works well.
2. If answer to the question 1 is yes, I guess all I need to do it to modify the test.py to make it work on non-OTB dataset, right?
3. If not, how hard do you think is to retrain or fine-tune your pre-trained model on KITTI monocular tracking? Rough estimation is perfectly fine, I just want to get an idea of the difficulty.

Thank you for the code and any help you can provide!

**Comment by amoudgl** [comment]
Hi, thanks for your comments! Following are my suggestions: 

1. Ideally, it should work fine since the object tracking task by definition is class agnostic. Training data also includes vehicle class (cars, bikes etc.).
2. Yes, you just need to write a new class similar in `test.py`. 
3. I think finetuning could yield some improvement but not sure, didn't try this.
